README.rst

```diff
@@ -3,4 +3,30 @@ The conservation element and solution element (CESE) method
 ===========================================================
 
 .. image:: https://mybinder.org/badge_logo.svg
-  :target: https://mybinder.org/v2/gh/yungyuc/turgon/master?filepath=notebook
\ No newline at end of file
+  :target: https://mybinder.org/v2/gh/yungyuc/turgon/master?filepath=notebook
+
+Build Turgon on OS X
+====================
+Clone from https://github.com/yungyuc/turgon.git:
```

Comment:
There should be a blank line between the above section mark '=====' and this line